netbox/dcim/tables/devices.py

```diff
@@ -53,6 +53,16 @@ def get_cabletermination_row_class(record):
     return ''
 
 
+def get_interface_row_class(record):
+    if not record.enabled:
+        return 'danger'
+    elif not record.is_connectable:
```

Comment:
```suggestion
    elif record.is_virtual:
```

This should evaluate `is_virtual` to accommodate wireless interfaces in v3.1.